doc/ref_cert/lfn/chapters/chapter05.md

```diff
@@ -231,7 +231,29 @@ It defines the end-end Certification Process required for certifying the given V
 using this platform (providing above requirements), producing the test results to
 certification portal , review and granting the badge to this VNF.
 
-[More details](https://wiki.lfnetworking.org/display/LN/VNF+Certification+Lifecycle)
+The complete process is detailed below:
+
+1. *VNF Vendors* submit the VNF into OVP Lab for certification testing_guide
+
+2. Using VNF Test Platform, VNF will be tested with all required test cases Provided
+by *Test authors* by using the appropriate MANO and NFVi provided by Vendors. while

```

Comment:
These are suppliers of VNFs/CNFs, and are seeking OPNFV Certification Badges for their product against NFVI certified RIs for the respective RA.  The title of this issue #807 we're commenting on is misleading, as this is specific to RC-1 Chapter 5, "VNF E2E C&V Framework Requirements".  Which is intended to be the Framework (of requirements) for VNF E2E badging.  Its assumed that the VNF supplier, or vendor, has ensured they have setup a lab and received NFVI Certification Badging before proceeding with their VNF testing.  

Line number 88 calls out the need for VNFs to test against RM/RA (I just added RA) certified platforms.  I have rewritten line 88 so that it's more clearly stated.  For convenience, here is the complete rewrite:  

_Suppliers of VNFs/CNFs seeking to receive VNF certification badges must first ensure their testing is performed against a compliant RM/RA architecture supporting all capabilities, features, and services defined by the respective *RM/RA requirements*.  More specifically, the VNF Supplier must ensure their implementation of the RM/RA receives the NFVI Certification badge prior to starting VNF testing.  Finally, to receive VNF Certification, the test platform will need to support TOSCA and HOT based VNF distros._

@lylavoie (Lincoln), please review and confirm if we can close (resolve) this thread.  Thanks.

Reply:
@lylavoie - can we close/resolve this conversation please? thanks

Reply:
I like the rewrite, I'll mark as resolved.